### DISCUSSION TITLE
How to improve download speed on bunkrr?

### DISCUSSION THREAD

**Discussion by dexter-amico** [jbsparrow/CyberDropDownloader#375] [Q&A] [answered]
There are some configs that would improve download speed when downloading from bunkrr? Usually it download rate is something about 15kbps and 150 kbps, when lucky it can get faster as 1 mbps and beyond, but it isn't the most common speed.

**Comment by NTFSvolume** [comment, marked as answer]
They have problems with their servers. 
- 1 server had a hardware failure (`taquito`). 
- 2 other servers lost their cache and that's why downloads are slow (`burger` and `milkshake`). These are their biggest servers, that's why basically every download is slow.

Just wait until they fix it.

**Reply by dexter-amico** [reply]
Thank you for the explanation, and please keep up with the good work!
